client/settings/express-checkout-settings/general-payment-request-button-settings.js

```diff
@@ -165,23 +166,41 @@ const GeneralPaymentRequestButtonSettings = ( { type } ) => {
 		isPaymentRequestEnabled &&
 		isWooPayFeatureFlagEnabled;
 
+	const checkoutPageUrl = `${ ADMIN_URL }post.php?post=${
+		getSetting( 'storePages' )?.checkout?.id
+	}&action=edit`;
+
 	return (
 		<CardBody>
 			{ showWarning && (
-				<InlineNotice
-					status="warning"
-					icon={ true }
-					isDismissible={ false }
-				>
-					{ sprintf(
-						/* translators: %s type of button to which the settings will be applied */
-						__(
-							'These settings will also apply to the %s on your store.',
-							'woocommerce-payments'
-						),
-						otherButtons
-					) }
-				</InlineNotice>
+				<>
+					<InlineNotice
+						status="warning"
+						icon={ true }
+						isDismissible={ false }
+					>
+						{ sprintf(
+							/* translators: %s type of button to which the settings will be applied */
+							__(
+								'These settings will also apply to the %s on your store.',
+								'woocommerce-payments'
+							),
+							otherButtons
+						) }
+					</InlineNotice>
+					<InlineNotice
+						status="warning"
+						icon={ true }
+						isDismissible={ false }
+					>
+						{ __(
+							'Some appearance settings may be overridden by the express payment section of the'
+						) }{ ' ' }
+						<a href={ checkoutPageUrl }>
+							{ __( 'Cart & Checkout blocks.' ) }
+						</a>
```

Comment:
I think we should add a `checkoutPageUrl` link to the word "checkout" and a `cartPageUrl` link to the word "cart." Also, we shouldn't render this notice at all if both are empty.